javaparser-core-generators/src/main/java/com/github/javaparser/generator/core/other/TokenKindGenerator.java

```diff
@@ -80,7 +80,7 @@ public void generate() {
     }
 
     private void generateValueOfEntry(SwitchStmt valueOfSwitch, String name, IntegerLiteralExpr kind) {
-        final SwitchEntry entry = new SwitchEntry(new NodeList<>(kind), SwitchEntry.Type.STATEMENT_GROUP, new NodeList<>(new ReturnStmt(name)));
+        final SwitchEntry entry = new SwitchEntry(new NodeList<>(kind), SwitchEntry.Type.STATEMENT_GROUP, new NodeList<>(new ReturnStmt(name)), false);
```

Comment:
Can you use the constructor without the 'default' argument to limit the impact on the code?

Reply:
There isn't a constructor without a `default` argument anymore, but I could add that if you prefer.

Reply:
Yes, because it seems to me that there is an update to the code that deals with this point. So it would avoid this change, and it would possibly have an impact on other projects based on Javaparser.